### DISCUSSION TITLE
Reset Password Link Gibberish

### DISCUSSION THREAD

**Discussion by techreative-gh** [directus/directus#17304] [Archive]
I'm hosting on linode via elest.io.  Have Directus connected to Supabase. I'm not using a .env file for configuration but instead docker-compose.yaml with an env to store passwords. I was testing the forgot password workflow. From Gmail, the Reset Your Password link comes as https://fgbdeif.r.bh.d.sendibt3.com/tr/cl/s6kcfar8QYotDdxqBr9tSVk1IjP3uhgbNeSvaSJ34qEGFQgfr3s9TldEdWAHCY33hnlkIjDj9V3WQ6O6V_sK627SO8kuuqsCTHeyN2GJJfV8eZk9NZ0G45tlSVwsJPytPd8DVND-xgfvQ7drHPbt_YfAAcnlXD6gSdwZky-Gt6XshX7ooXUhtTc68n0vO_aRTTtAsHbmzASpT-46APgfeEz0sZPHGGht60eOJ2msZriDNL20evfOTKbp8dM60-1kzZ6XQD6WoLGxJ8izX4NpDd1FnKnlkm1RC_qWgyP4Nb-UikBhoxsovg86z5O_kJP3nvkpaM08Iv9_yythzivlvWOAn5JzXwSsEQOo799e_kuEIvumGYu07YtJFU2enn8OMRGj2Z7VuDq0Rw-QTXmDghnL7gfLcXATmF3GPPMuPeffPl9n35YSQ_HATR2XI-TGUy-La2MNDGXKIRvYImviUBzmPq57Y2eHttA_cU5NeEzyl5lJPJkuV2rF1ItUs-RpTonsyqLet1Kx0jbM4y9-9KHoJGBoepvQRiGIbwmjhiY4vSsSqWSz998zPBU

When I dig deeper into Sendinblue, the logs show the original email had href="/admin/reset-password?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InRlY2hyZWF0aXZlbGxjQGdtYWlsLmNvbSIsInNjb3BlIjoicGFzc3dvcmQtcmVzZXQiLCJoYXNoIjoiNmNkZjY3YyIsImlhdCI6MTY3NDY3NjE1NCwiZXhwIjoxNjc0NzYyNTU0LCJpc3MiOiJkaXJlY3R1cyJ9.MfmrAk8N7PvC84ErCQBVkO2-Qu8FvzSrTegNKlrDol4" as the link.

Do we know what could be causing this? How do I get the base url in the emails?

**Comment by rijkvanzanten** [comment]
You're missing the config for `PUBLIC_URL`. Other than that, the `sendibt3` stuff is a link tracking thing from Sendinblue I think

**Reply by techreative-gh** [reply]
Absolutely right, thanks for the quick reply. This setup was throwing me off a bit as I haven't messed with docker-compose, but I just dropped the PUBLIC_URL in the .yml file and it took off.  I do have to recommend elest.io with Linode.  Builds are lightning quick.  I just tried fly.io and the build times were slow and I just couldn't get things working right.  Looking forward to digging into this more.
